host/sgx/sgxquoteprovider.c

```diff
@@ -292,4 +349,14 @@ void oe_free_sgx_quote_verification_collateral_args(
         if (args->host_out_buffer)
             free(args->host_out_buffer);
     }
+}
+
+void oe_free_sgx_quote_verification_collateral_with_params_args(
+    oe_get_sgx_quote_verification_collateral_with_params_args_t* args)
+{
+    if (args)
+    {
+        if (args->host_out_buffer)
```

Comment:
Delete line 359, free() is defined as being safe to call with null, and it reduces code complexity and potentially increases test coverage to delete the line.